src/applications/disability-benefits/all-claims/content/instructionalPart1.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const instructionalPart1Description = (
```

Comment:
Should this be a default export, if it's the only thing you expect to be in this file?

Reply:
changed. Thank you!